core/src/main/java/fr/sncf/osrd/envelope_sim/allowances/Allowance.java

```diff
@@ -0,0 +1,7 @@
+package fr.sncf.osrd.envelope_sim.allowances;
+
+import fr.sncf.osrd.envelope.Envelope;
+
+public interface Allowance {
+    Envelope apply(Envelope base, AllowanceType type, double value);
```

Comment:
It's never used and the only override does nothing, will it be used later on?

Reply:
It will be used by flo very very soon, it was updated already